bot/potentialScammer/fetchTargetChannel.module.js

```diff
@@ -0,0 +1,34 @@
+//because the bot will tell the user who may be scammer
+//so the bot needs to send the a list of scammers on 'may-be-scammer' channel
+//find this channel on all channels,and tell everyone who might be a scammer
+
+const getChannels = (guild) => {
+  return guild.channels.cache
+}
+
+const isTargetChannel = (channel) => {
+  return (channel.name === 'may-be-scammer' && channel.type === 'GUILD_TEXT')
+}
+
+const searchTargetChannel = (channels, targetChannelRules) => {
+  return channels.find(channel => targetChannelRules(channel))
+}
+
+const fetchTargetChannel = async (guild) => {
+  const channels = getChannels(guild)
+  let targetChannel = searchTargetChannel(channels, isTargetChannel)
+  if (!targetChannel) {
+    targetChannel = guild.channels.create('may-be-scammer', {
```

Comment:
According to SAL http://principles-wiki.net/principles:single_level_of_abstraction#:~:text=All%20statements%20of%20a%20method,will%20result%20in%20smaller%20methods.
Extract the code in if block as `createTargetChannel()` function.